tools/testsys/src/crds.rs

```diff
@@ -41,7 +41,7 @@ impl<'a> CrdInput<'a> {
                 metadata_base_url, targets_url
             );
             Some(TufRepoConfig {
-                metadata_url: format!("{}{}/{}", metadata_base_url, &self.variant, &self.arch),
+                metadata_url: format!("{}{}/{}/", metadata_base_url, &self.variant, &self.arch),
```

Comment:
Is the last `/` needed? What changed?

Reply:
It is needed for `tuftool`. Now that we are copying OVAs with `tuftool`, the extra `/` is needed.